src/libs/actions/connections/QuickbooksOnline.ts

```diff
@@ -21,7 +22,11 @@ function getQuickbooksOnlineSetupLink(policyID: string) {
     return commandURL + new URLSearchParams(params).toString();
 }
 
-function updateQuickbooksOnlineEnableNewCategories(policyID: string, settingValue: boolean) {
+function buildOnyxDataForMultipleQuickbooksConfigurations<TConfigUpdate extends Partial<Connections['quickbooksOnline']['config']>>(
```

Comment:
Nice, I see you were able to avoid a lot of code repetition by creating this helper method